source/_docs/ecosystem/caddy.markdown

```diff
@@ -10,12 +10,40 @@ footer: true
 ---
 
 Configure [Caddy Server](https://caddyserver.com/) for use as a reverse proxy to Home Assistant.
+## Install
 
+<p class='note warning'>
+It can potentially be dangerous to send a command straight to bash please make sure you trust the source before running that type of commands, to check what this command does head over to [getcaddy.com](https://getcaddy.com/)
+</p>
+
+Once you are ready to proceed with the installer type:
+```
+curl https://getcaddy.com | bash -s personal
 ```
-hass.example.org {
-    proxy / localhost:8123 {
+
+ 
+Then create a file named ``Caddyfile``:
+```
+hass.yourdomain.org {
+    proxy / yourhassinternalip:8123 {
         websocket
         transparent
     }
 }
 ```
+
+Make sure to forward ports: `80` and `443` on your router so that the SSL certificates can be generated
+
+To run your Caddy Server type:
+```
+caddy -agree=true -conf=/path/to/your/Caddyfile -email youremail@example.com
+```
+If you encounter issues with Google TTS update your ``base_url`` to include ``https``:
+```yaml
+http:
+  base_url: https://hass.example.org
+```
+## More Info
```

Comment:
Same as with previous title, and an empty line below as well.

Reply:
done